tests/Sulu/Component/Content/Mapper/ContentMapperTest.php

```diff
@@ -2299,6 +2299,60 @@ public function testMultipleLanguagesCopy()
         $this->assertEquals('/page-1', $result->url);
     }
 
+    private function prepareCopyLanguageTree()
+    {
+        $this->mapper->saveStartPage(array('title' => 'Start Page'), 'overview', 'default', 'de', 1);
+        $this->mapper->saveStartPage(array('title' => 'Start Page'), 'overview', 'default', 'en', 1);
+
+        $data = array(
+            array(
+                'title' => 'test',
+                'url' => '/test'
+            ),
+            array(
+                'title' => 'childtest',
+                'url' => '/test/childtest'
+            )
+        );
+
+        $data[0] = $this->mapper->save($data[0], 'overview', 'default', 'de', 1);
+        $data[1] = $this->mapper->save($data[1], 'overview', 'default', 'de', 1, true, null, $data[0]->getUuid());
+
+        return $data;
+    }
+
+    public function testCopyLanguageTree()
+    {
+        $data = $this->prepareCopyLanguageTree();
+
+        $this->mapper->copyLanguage($data[0]->getUuid(), 1, 'default', 'de', 'en');
+        $this->mapper->save(
+            array('title' => 'test-en', 'url' => '/test-en'),
+            'overview',
+            'default',
+            'en',
+            1,
+            true,
+            $data[0]->getUuid(),
+            null,
+            null,
+            false
+        );
```

Comment:
Does this test cover two separate cases?

If so, it should be two separate test cases.